changelogs/fragments/1040-ldap_search-changed-must-be-false.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - ldap_search - replaced the output value ``changed`` with ``False`` (https://github.com/ansible-collections/community.general/issues/1040).
```

Comment:
```suggestion
  - ldap_search - the module no longer reports a change (https://github.com/ansible-collections/community.general/issues/1040).
```

Reply:
Well, maybe "the module no longer lies about having a change to report" :-)

Reply:
`no longer incorrectly reports a change`?